docs/reference/content/driver/getting-started/quick-start-pojo.md

```diff
@@ -142,6 +142,8 @@ CodecRegistry pojoCodecRegistry = fromRegistries(MongoClient.getDefaultCodecRegi
                 fromProviders(PojoCodecProvider.builder().automatic(true).build()));
 ```
 
+Note: The registries are checked in order until one returns a Codec for the requested class.
```

Comment:
Let's note here that the PojoCodecProvider should be last in the list.  I don't think that's obvious even with this note.  Perhaps:

   Because the `PojoCodecProvider` can provide a `Codec` for almost any class, including ones that are in the default registry, the `PojoCodecProvider` should always be the last `CodecProvider` in the list.